openpype/plugins/publish/extract_thumbnail.py

```diff
@@ -172,24 +183,28 @@ def _get_filtered_repres(self, instance):
             filtered_repres.append(repre)
         return filtered_repres
 
-    def create_thumbnail_oiio(self, src_path, dst_path):
+    def create_thumbnail_oiio(
+        self,
+        src_path,
+        dst_path,
+        config_path,
+        source_colorspace,
+        display,
+        view
+    ):
         self.log.info("Extracting thumbnail {}".format(dst_path))
-        oiio_tool_path = get_oiio_tools_path()
-        oiio_cmd = [
-            oiio_tool_path,
-            "-a", src_path,
```

Comment:
❤️ Nice. The removal of the `-a` flag for thumbnails also makes much more sense reading the docs:
> Using the `-a` command tells oiiotool to try to preserve all subimages from the inputs and apply all computations to all subimages: